### ISSUE TITLE
Unable to find _deps property in browser

### ISSUE THREAD

**Issue by alexlibby** [ractivejs/ractive#2972]
I've been reading about how Ractive manages its dependencies, from the docs at http://docs.ractivejs.org/latest/dependants. I had a look for this in console within a sample Ractive app I have built, and can't seem to find this property anywhere - I'm using the current 0.8 release of Ractive, in Google Chrome 58.

I had a look on Stack Overflow, and found this article from 2015: https://stackoverflow.com/questions/33072526/list-of-dependencies-of-a-ractive-instance - this suggested that the property is out of date / doesn't exist. If this is the case, then should the docs article not be updated? If it does need updating, what should be there now - does the current version of Ractive still have a similar property available, where we can see how Ractive manages dependencies please?

**Comment by evs-chris** [comment]
Yeah, that's way out of date. The docs are also moving to ractive.js.org, which is located in the ractivejs/ractivejs.github.io repo.

Deps as of 0.8 are tracked per keypath/model. Starting with ractive.viewmodel, each child model has a deps array containing references to vdom nodes, observers, etc that depend on the keypath. When a model changes, it notifies it's deps and then its children to notify their deps and so on.

**Comment by alexlibby** [comment]
Ah OK - that makes sense now: thanks for confirming it ;-)

I've tried entering ```ractive.viewmodel``` into the console - I can indeed see entries there (such as "DOMStringMap"), and these begin to tie up with what's in my code. I know this is something that people wouldn't normally need to get involved in, but perhaps a page that outlines some of the entries and what they mean, might be worthwhile? It would be a case of having to set expectations as to what people can and can't do, but it might be interesting to know more about how this relates to one's code, perhaps?

EDIT: I've just checked https://ractive.js.org/concepts/data-binding/#dependents - I'm a bit confused: it looks like this very similar to the current version of the documentation? I assume this is still to be updated, right?
